### DISCUSSION TITLE
migrating to a noisepy/noisepy repository

### DISCUSSION THREAD

**Discussion by mdenolle** [mdenolle/NoisePy#42] [General]
We are looking to improve the ways to contribute to the codes and migrating to a noisepy/noisepy repository. Please comment below if you are interested in joining the team of contributors.

**Comment by ahmadtourei** [comment]
Hi Marine! I'm interested! Could you please provide more information on it?

**Comment by mdenolle** [comment]
We made our major first upgrade! Start new discussions for more specific goals for future contributions.
